quadriga/src/main/java/edu/asu/diging/quadriga/core/service/ICollectionManager.java

```diff
@@ -0,0 +1,45 @@
+package edu.asu.diging.quadriga.core.service;
+
+import java.util.List;
+
+import edu.asu.diging.quadriga.core.exceptions.CollectionNotFoundException;
+import edu.asu.diging.quadriga.core.model.Collection;
+
+public interface ICollectionManager {
```

Comment:
Is there a reason this has been renamed? It looks like the other service classes follow the convention interface does not start with `I` and implementation classes end with `Impl`.

Reply:
Yeah. I got confused with the naming convention from the other project. I'll change it back.